apps/console/src/features/secrets/pages/secret-edit.tsx

```diff
@@ -55,7 +51,7 @@ const SecretEdit: FC<SecretEditProps> = (props: SecretEditProps): ReactElement =
     const [ loading, setLoading ] = useState<boolean>(true);
     const [ resourceNotFound, setResourceNotFound ] = useState<boolean>(false);
     const [ editingSecret, setEditingSecret ] = useState<SecretModel>(undefined);
-    const [ secretType, setSecretType ] = useState<string>();
+    const [ , setSecretType ] = useState<string>();
```

Comment:
Do we need this state if we are not going to use it anywhere?

Reply:
We could remove this since it's only being used @ line #65.

Reply:
Addressed with commit 3393425